VAMobile/src/components/VAButton.test.tsx

```diff
@@ -32,137 +22,81 @@ context('VAButton', () => {
         disabledText={'my button instructions'}
       />,
     )
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
+  it('initializes correctly', () => {
+    expect(screen.getByRole('button', { name: 'my button' })).toBeTruthy()
   })
 
-  it('should call onChange', async () => {
-    testInstance.findByType(VAButton).props.onPress()
+  it('should call onChange', () => {
+    fireEvent.press(screen.getByRole('button'))
     expect(onPressSpy).toBeCalled()
   })
 
   describe('when icon props are passed in', () => {
-    it('should render a VAIcon', async () => {
+    it('should render a VAIcon', () => {
       initializeTestInstance(false, ButtonTypesConstants.buttonPrimary, true)
-      expect(testInstance.findAllByType(VAIcon).length).toEqual(1)
+      expect(screen.UNSAFE_getAllByType(VAIcon).length).toEqual(1)
     })
   })
 
   describe('when disabled is true', () => {
-    it('should set the text color to "buttonDisabled"', async () => {
+    beforeEach(() => {
       initializeTestInstance(true)
-      expect(testInstance.findAllByType(TextView)[0].props.color).toEqual('buttonDisabled')
     })
-
-    it('should set the background color to "buttonDisabled"', async () => {
-      initializeTestInstance(true)
-      expect(testInstance.findAllByType(Box)[0].props.backgroundColor).toEqual('buttonDisabled')
+    it('should set the text color to "buttonDisabled"', () => {
+      expect(screen.UNSAFE_getByProps({ color: 'buttonDisabled' })).toBeTruthy()
     })
 
-    it('should set the border color to "undefined"', async () => {
-      initializeTestInstance(true)
-      expect(testInstance.findAllByType(Box)[0].props.borderColor).toEqual(undefined)
+    it('should set the background color to "buttonDisabled"', () => {
+      expect(screen.UNSAFE_getByProps({ backgroundColor: 'buttonDisabled' })).toBeTruthy()
     })
 
-    it('should show the disabled message', async () => {
-      initializeTestInstance(true)
-      expect(testInstance.findAllByType(TextView)[1].props.children).toEqual('my button instructions')
+    it('should show the disabled message', () => {
+      expect(screen.getByText('my button instructions')).toBeTruthy()
+    })
+    it('should show accessibility state of disabled', () => {
+      expect(screen.getByAccessibilityState({ disabled: true })).toBeTruthy()
```

Comment:
I also believe disabled buttons are no longer going to be a thing too so this test can be removed as well, even though when the conversion goes through this file and VAButton will be removed from the app

Reply:
Left the disabled test in for now, we can remove it once the conversion is completed (and the entire file is deleted)